dll/shellext/shellbtrfs/recv.cpp

```diff
@@ -27,6 +27,7 @@
 
 
 #ifndef _MSC_VER
+#define __cpuidex __cpuidex_ // prevent redeclaration
```

Comment:
```suggestion
#ifdef __REACTOS__
#define __cpuidex __cpuidex_ // prevent redeclaration
#endif
```

Reply:
@SergeGautherie I am not sure about that, but I could imagine that the `#ifndef _MSC_VER` one line above had also been introduced by Reactos developers.

Reply:
No. Moreover, if it had, then the whole block would need to be marked as such.

Reply:
Confirmed.
![image](https://github.com/reactos/reactos/assets/33393466/e48e9d05-062d-4abf-b303-af34b69b6461)
Upvoting your proposal therefore.